Completed css-two-column-layout challenge. (hopefully)

joshangeles opened the pull request:
![image](https://user-images.githubusercontent.com/68949703/206791558-424cb6bf-0df3-4db7-a9dd-1ab635c5730a.png)

codingcodymiller commented on css-two-column-layout/styles.css:
```diff
@@ -0,0 +1,85 @@
+}
+
+.col-fourth {
+  width: 25%;
+  text-align: right;
+  padding-right: 0;
```
`column` classes should only ever have `flex-basis`, nothing else. Anything additional necessary should be on a separate class.

codingcodymiller commented on css-two-column-layout/styles.css:
```diff
@@ -0,0 +1,85 @@
+}
+
+.col-three-fourths {
+  width: 75%;
+  align-items: right;
+  text-align: left;
```
`column` classes should only ever have `flex-basis`, nothing else. Anything additional necessary should be on a separate class.

codingcodymiller commented on css-two-column-layout/styles.css:
```diff
@@ -0,0 +1,85 @@
+  font-size: larger;
+}
+
+#learnmore {
+  margin-left: 425px;
```
😱 

There is a better way to do this, I'm sure. `text-align` typically is plenty to center a button element

joshangeles commented:
![image](https://user-images.githubusercontent.com/68949703/207192763-1a6e51fa-d8c2-49d2-babe-3f0100916a4b.png)

Removed unnecessary rules from column rulesets and changed the learn more button to not use a gross amount of margin.
